app/frontend/src/utils/commonServices.js

```diff
@@ -50,7 +50,7 @@ export const CommonServiceList = Object.freeze([{
   type: kc,
   shortName: 'common-document-generation',
   name: 'Common Document Generation Service',
-  apiDocLink: 'https://cdogs-master-idcqvl-prod.pathfinder.gov.bc.ca/api/v1/docs',
+  apiDocLink: 'https://cdogs-master-idcqvl-dev.pathfinder.gov.bc.ca/api/v2/docs',
```

Comment:
this should be a link to the PROD api docs but I believe it's currently not public.
I'll change this if possible